app/config.py

```
@@ -0,0 +1,89 @@
+
+        return headers
+
+
+@lru_cache
+def get_settings() -> Settings:
+    """Get cached settings instance"""
+    return Settings()
+
+
+class _SettingsProxy:
+    def __getattr__(self, name: str) -> Any:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)